syntax = "proto3";

package scoresync;

message MakeRankRequest { 
    string input_json_path = 1;
    string output_json_path = 2;
}

message MakeRankResponse {}

message CleanDataRequest { 
    string input_json_path = 1;
    string output_json_path = 2;
}
message CleanDataResponse {}

message AddDetailsRequest { 
    string input_json_path = 1;
    string output_json_path = 2;
    string old_json_path = 3;
}
message AddDetailsResponse {}

service ScoreSyncService {
    rpc MakeRank(MakeRankRequest) returns (MakeRankResponse);
    rpc CleanData(CleanDataRequest) returns (CleanDataResponse);
    rpc AddDetails(AddDetailsRequest) returns (AddDetailsResponse);
}
